### PR TITLE
Fix CI/CD

### PR DESCRIPTION
**Pull request by itsksaurabh**
# Description

Please provide a detailed description of what was done in this PR

# Changes

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)
- [ ] Changes only for a subset of nodes

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it

# Nodes audience

In case this PR includes changes that must be applied only to a subset of nodes, please specify how you handled it (e.g. by adding a flag with a default value...)

# Checklist

- [ ] I have added at least 2 reviewer or the whole pos-v1 team
- [ ] I have added sufficient documentation in code
- [ ] I will be resolving comments - if any - by pushing each fix in a separate commit and linking the commit hash in the comment reply
- [ ] Created a task in Jira and informed the team for implementation in Erigon client (if applicable)
- [ ] Includes RPC methods changes, and the Notion documentation has been updated

# Cross repository changes

- [ ] This PR requires changes to heimdall
    - In case link the PR here:
- [ ] This PR requires changes to matic-cli
    - In case link the PR here:

## Testing

- [ ] I have added unit tests
- [ ] I have added tests to CI
- [ ] I have tested this code manually on local environment
- [ ] I have tested this code manually on remote devnet using express-cli
- [ ] I have tested this code manually on mumbai
- [ ] I have created new e2e tests into express-cli

### Manual tests

Please complete this section with the steps you performed if you ran manual tests for this functionality, otherwise delete it

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it

### CODE REVIEW

File: .github/matic-cli-config.yml [hunk 1]
```diff
@@ -20,6 +21,7 @@ ethURL: http://ganache:9545
 ethHostUser: ubuntu
 devnetType: docker
 borDockerBuildContext: "../../bor"
-heimdallDockerBuildContext: "https://github.com/maticnetwork/heimdall.git#develop"
+heimdallDockerBuildContext: "https://github.com/shibaone/heimdall.git#develop"
 sprintSizeBlockNumber:
   - '0'
+# devnetBorFlags: config,config,config
```
Comment (ShibWizard) [inline .github/matic-cli-config.yml]: Why this is commented ?

File: .golangci.yml [hunk 2]
```diff
@@ -185,4 +185,4 @@ issues:
   max-issues-per-linter: 0
   max-same-issues: 0
   #new: true
-  new-from-rev: origin/master
+  new-from-rev: origin/v1.1.0-bone-candidate
```
Comment (ShibWizard) [inline .golangci.yml]: Can we revert this ?

File: .github/matic-cli-config.yml (hunk 1 above)
Comment (itsksaurabh) [reply .github/matic-cli-config.yml]: This line was not there in the config but I added it based on how matic cli config is being used by https://github.com/maticnetwork/bor. So I later commented it out.

File: .golangci.yml (hunk 2 above)
Comment (itsksaurabh) [reply .golangci.yml]: sure, I missed it sorry.

**Review by ShibWizard — DISMISSED** [review]

**Pull request merged**